spec/models/preservation_copy_spec.rb

```diff
@@ -12,7 +12,7 @@
     )
   end
   let!(:preservation_policy) do
-    PreservationPolicy.create!(preservation_policy_name: 'default',
+    PreservationPolicy.create!(preservation_policy_name: 'pc_spec_default',
                                archive_ttl: 604_800,
                                fixity_ttl: 604_800)
```

Comment:
I imagine we should use the default policy from the db, not create one ... will that fix it?